cartofacade/carto_facade.go

```diff
@@ -173,3 +173,126 @@ func (cf *CartoFacade) start(ctx context.Context, activeBackgroundWorkers *sync.
 		}
 	}()
 }
+
+// Initialize calls into the cartofacade C code.
+func (cf *CartoFacade) Initialize(ctx context.Context, timeout time.Duration, activeBackgroundWorkers *sync.WaitGroup) error {
```

Comment:
Can we please move all these public functions to the top of the file so that the package users can read all the public functions & types we expect them to use before needing to read any private ones?